sky/clouds/aws.py

```diff
@@ -5,6 +5,9 @@
 import typing
 from typing import Dict, Iterator, List, Optional, Tuple
 
+import boto3
+import botocore
```

Comment:
Do we want to move this import into the `check_credentials()` with `try... except` and logs saying `'boto3 is not installed properly. Please run: pip install boto3`

Reply:
Done, thanks!